internal/xcrd/schemas.go

```diff
@@ -230,7 +230,7 @@ func CompositeResourceClaimPrinterColumns() []extv1.CustomResourceColumnDefiniti
 		},
 		{
 			Name:     "AGE",
-			Type:     "Date",
+			Type:     "date",
```

Comment:
wasn't sure why this changed but I assume you know what is happening :)

Reply:
See https://github.com/crossplane/crossplane/pull/2001/commits/7150d363326736a44fb8291aec40f2d12700308b